app/join/form-state.ts

```diff
@@ -7,12 +8,15 @@ export type SubmissionState =
   | { tag: "submitted_successfully" }
   | { tag: "submission_error"; msg: string };
 
+type PrivacyFlags = (typeof privacyFlags)[number][];
```

Comment:
Nechceš ten typ přesunout do `src/data/user-profile`, ať je to pohromadě?

Reply:
Ups, myslel jsem, že to ještě stihnu dodělat a sjednotit copy s Laurou, ale ty jsi akční a už jsi to zamergoval. Nevadí, příště. 😀

Reply:
Haha! Pardon. Klidně to měň přímo v `topic/profiles`, jsem s těmi force pushi opatrný :)